test/drive.js

```diff
@@ -128,6 +149,41 @@ test('Multiple Archives File', function (t) {
   })
 })
 
+test('Multiple Archive POST File', function (t) {
+  var onrequest = hyperdriveHttp(getArchive)
+  server.once('request', onrequest)
+  fs.createReadStream(path.join(__dirname, 'drive.js'))
+  .pipe(request.post('http://localhost:8000/', function (err, res, body) {
```

Comment:
yeah, in the multi archive case, now the `getArchive` function is able to return an archive even when there is no `datUrl.key` provided